pkg/status/status.go

```diff
@@ -0,0 +1,155 @@
+package status
+
+import (
+	configv1 "github.com/openshift/api/config/v1"
+	configclient "github.com/openshift/client-go/config/clientset/versioned/typed/config/v1"
+	cohelpers "github.com/openshift/library-go/pkg/config/clusteroperator/v1helpers"
+	operatorhelpers "github.com/openshift/library-go/pkg/operator/v1helpers"
+	"github.com/operator-framework/operator-marketplace/version"
+	log "github.com/sirupsen/logrus"
+	"k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/api/meta"
+	"k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const clusterOperatorName = "marketplace-operator"
+
+type status struct {
+	client          *configclient.ConfigV1Client
+	coAPINotPresent bool
+	namespace       string
+	co              *configv1.ClusterOperator
```

Comment:
Can we have a better name for this? I think `coAPINotPresent` is fine for the purpose of brevity, but I think it would help readability if this variable was called something like `clusterOperator`.

Reply:
Done